modules/DMZ/data_kit/data_splitting.py

```diff
@@ -1,10 +1,18 @@
-from modules.toolbox import *
 import pandas as pd
 from sklearn import preprocessing
 from sklearn import cross_validation
-
-
-# Helper functions for the whole framework to use
+import numpy as np
+
+
+# Gets the training and testing splits for training.
```

Comment:
The usage documentation for a method should be in the folder wherever the user imports the code from.  So in this case I would think `modules/DMZ/data_kit` would be a good place for usage documentation.